src/main/java/com/dataloom/authorization/Principals.java

```diff
@@ -30,7 +30,7 @@ public static Principal getCurrentUser() {
         return new Principal(
                 PrincipalType.USER,
                 ( (Auth0UserDetails) SecurityContextHolder.getContext().getAuthentication().getPrincipal() )
-                        .getUsername() );
+                        .getAuth0Attribute("user_id").toString() );
```

Comment:
Just wondering - did you test this? I will look at this and try this out very soon, but I vaguely remember that getting "user_id" always return null for me in the past.